src/index.js

```diff
@@ -0,0 +1,137 @@
+document.querySelector("#tempUp").onclick = function () {
+    const temp = document.querySelector("#tempVisual");
+    temp.textContent++;
+    changeTempAndLand(temp.innerHTML);
+};
+
+document.querySelector("#tempDown").onclick = function () {
+    const temp = document.querySelector("#tempVisual");
+    temp.textContent--;
+    changeTempAndLand(temp.innerHTML);
+};
+
+const changeTempAndLand = (temp) => {
+    let color = 'green';
+    let landscape = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+
+    const tempColor = document.querySelector("#tempVisual");
+    const landscapeType = document.getElementById("landscape");
+
+    if (temp > 80) {
+        color = 'red';
+        landscape = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂';
+    } else if (temp > 70) {
+        color = 'orange';
+        landscape = '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷';
+    } else if (temp > 60) {
+        color = 'yellow';
+        landscape = '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃';
+    } else if (temp > 50) {
+        color = 'green'
+        landscape = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+    } else {
+        color = 'teal';
+        landscape = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+    }
+
+    tempColor.style.color = color;
+    landscapeType.textContent = landscape;
+};
+
+const state = {
+    city: 'Los Angeles',
+    lat: 34.0536909,
+    lon: -118.242766,
+    temp: 53
+};
+
+const newCity = document.getElementById("cityNameInput");
+
+state.city = String(newCity.innerHTML);
+
+const changeCityBtn = document.getElementById("search");
+const resetBtn = document.getElementById("cityReset")
+const cityTitle = document.getElementById("headerCity");
+
+
+const changeCityByInput = () => {
+    cityTitle.innerHTML = newCity.value;
+};
+
+
+// changeCityByInput();
+
+changeCityBtn.addEventListener("click", changeCityByInput);
```

Comment:
There's an issue with wave 03 (naming the city), the requirements specify that there should be an element that displays a city name & a text input element that allows the user to change the city name. The city name must also be updated every time there's a change to the text input element. Currently your code does not chnage the displayed city name that reflects what's being typed in the input box.